chef-dk.gemspec

```diff
@@ -44,8 +44,10 @@ Gem::Specification.new do |gem|
   gem.add_dependency "ffi-yajl", ">= 1.0", "< 3.0"
   gem.add_dependency "minitar", "~> 0.5.4"
   gem.add_dependency "chef", "~> 12.5"
-  gem.add_dependency "solve", ">= 2.0.1"
+  gem.add_dependency "solve", "~> 2.0", ">= 2.0.1"
+  gem.add_dependency "addressable", "~> 2.3.4"
   gem.add_dependency "cookbook-omnifetch", "~> 0.2", ">= 0.2.2"
+  gem.add_dependency "addressable", "~> 2.3.4"
```

Comment:
Travis is currently failing because this duplicates line 48 and bundler doesn't like that.